cli/export/export.go

```diff
@@ -0,0 +1,76 @@
+/*
+Tool to export annotation results from the internal DB to an output DB.
+
+Usage: export <path-to-origin.db> <path-to-destination.db>
+*/
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/jessevdk/go-flags"
+)
+
+const desc = `Exports annotation results from the internal input database to a new output
+database.
+The destination file must not exist.`
+
+var opts struct {
+	Args struct {
+		Input  string `description:"SQLite database filepath"`
+		Output string `description:"SQLite database filepath"`
+	} `positional-args:"yes" required:"yes"`
+}
+
+func main() {
+	parser := flags.NewParser(&opts, flags.Default)
+	parser.LongDescription = desc
+
+	if _, err := parser.Parse(); err != nil {
+		if err, ok := err.(*flags.Error); ok {
+			if err.Type == flags.ErrHelp {
+				os.Exit(0)
+			}
+
+			fmt.Println()
+			parser.WriteHelp(os.Stdout)
+		}
+
+		os.Exit(1)
+	}
+
+	if _, err := os.Stat(opts.Args.Input); os.IsNotExist(err) {
```

Comment:
I think this is not neccesary, because when in [`if copy() != nil`](https://github.com/src-d/code-annotation/pull/15/files#diff-fe387ed169ad94895f58f2465e5359cdR53) it's already validated that there is no errors because the source can not be [opened](https://github.com/src-d/code-annotation/pull/15/files#diff-fe387ed169ad94895f58f2465e5359cdR59) or [readed](https://github.com/src-d/code-annotation/pull/15/files#diff-fe387ed169ad94895f58f2465e5359cdR71)

Reply:
True. But this is here in preparation for when the command uses this path to open an SQLite connection. This check will avoid creating a new file.

Reply:
fair enough